Source/WEB_ROOT/admin/faculty/asset/editAsset_not_needed.html

```diff
@@ -0,0 +1,460 @@
+<form action="" method="POST" id="assetform">
+~[DirectTable.Select:u_assettable;id:~(gpv.assetid)]
+	<div class="box-round">
+		<table class="grid">
+			<tr>
+				<td class="bold" width="15%">Active Status</td>
+				<td>
+					<select name="[u_assettable]active">
+						~[if.~(gpv.assetid)=-1]
+					        <option value="0">Inactive</option>
+							<option value="1" selected>Active</option>
+							<option value="2">Lost</option>
+							<option value="3">Recycled</option>
+						[else]
+		                    <option value="0" ~[if#sel.~([u_assettable]active)0]selected[/if#sel]>Inactive</option>
+                            <option value="1" ~[if#sel.~([u_assettable]active)1]selected[/if#sel]>Active</option>
+                            <option value="2" ~[if#sel.~([u_assettable]active)2]selected[/if#sel]>Lost</option>
+                            <option value="3" ~[if#sel.~([u_assettable]active)3]selected[/if#sel]>Recycled</option>
+								
+						[/if]
+					</select>
+				</td>
+			</tr>
+			<tr>
+				<td class="bold">School Name</td>
+				<td>
+					<select id="schoolDropDown">
+					~[if.~(gpv.assetid)=-1]
+						<option value="-1">Select a school</option>
+					[else]
+						~[tlist_sql;
+							SELECT schoolid,

```

Comment:
John, given the system you're working in I know you have limitations. Do they provide you with any facility to pull the queries out of the html? Can you create small stored procedure like programming elements allowing you to provide a better separation between view and backend concerns?